.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: php
```

Comment:
Could we please use GitHub Actions instead? Please check Enlighten or my Laravel component course.